tests/test_browser_model.py

```diff
@@ -1,6 +1,13 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
+import pytest
+
+
+@pytest.fixture
+def firefox_options(firefox_options):
+    firefox_options.set_preference('privacy.trackingprotection.enabled', True)
```

Comment:
Since pytest-selenium 0.11.0 you should be able to set this on tests that require it using:

```python
@pytest.mark.firefox_preferences({'privacy.trackingprotection.enabled': True})
```